src/components/ui/input-field/index.tsx

```diff
@@ -22,11 +23,26 @@ interface Props extends Partial<FieldProps> {
 
 export default function InputField(props: Props) {
   const classes = useClasses();
-  const { name, type, placeholder, contentBefore, contentAfter } = props;
-  const { register } = useFormContext();
+  const { message, name, type, placeholder, contentBefore, contentAfter } = props;
+  const { register, getFieldState } = useFormContext();
+
+  const getValidationState = () => {
+    if (message) return 'error';
+    if (getFieldState(name).isDirty) return 'success';
+    return 'none';
+  };
+
+  const getValidationMessage = () => {
+    if (message) return message;
+  };
```

Comment:
```suggestion
export default function InputField(props: Props) {
  const classes = useClasses();
  const { message, name } = props;
  const { register, getFieldState } = useFormContext();

  const getValidationState = () => {
    if (message) return 'error';
    if (getFieldState(name).isDirty) return 'success';
    return 'none';
  };
```
if you take input props as a basis for props, you can shorten the code